src/Transpiler.ts

```diff
@@ -339,7 +339,7 @@ export abstract class LuaTranspiler {
         this.importLuaLibFeature(func);
         params = params.filter(element => {
             return element.toString() !== "";
```

Comment:
Since there are changes here anyway, could you change this to `element => element.toString() !== ""`, it bothers me more than it should :D